sql/nm_spawn_points.sql

```diff
@@ -5660,6 +5660,10 @@ INSERT INTO `nm_spawn_points` VALUES (17498522,46,-150.407,48.460,-333.947);
 INSERT INTO `nm_spawn_points` VALUES (17498522,47,-138.264,48.466,-341.724);
 INSERT INTO `nm_spawn_points` VALUES (17498522,48,-130.664,48.315,-338.463);
 INSERT INTO `nm_spawn_points` VALUES (17498522,49,-164.707,48.330,-330.263);
+INSERT INTO `nm_spawn_points` VALUES (17502570,0,-196.187,0.51,396.813); -- Zipacna
+INSERT INTO `nm_spawn_points` VALUES (17502570,1,-121.073,16.575,420.915); -- Zipacna
+INSERT INTO `nm_spawn_points` VALUES (17502570,2,191.685, 0.289, 399.782); -- Zipacna
+INSERT INTO `nm_spawn_points` VALUES (17502570,3,141.353,16.575,440.782); -- Zipacna
```

Comment:
I don't know how this system works, will it just pick randomly from the points it has listed?

Reply:
That is my understanding, yes. Prior to having `INSERT INTO `nm_spawn_points` VALUES (17502570,0,-196.187,0.51,396.813); -- Zipacna` and
`UpdateNMSpawnPoint(mob:getID())` - it would spawn at the same point that was listed in mob_spawn_points.sql - now it randomly picks between the 4 locations.